tests/N98/Magento/Command/Cache/ReportCommandTest.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace N98\Magento\Command\Cache;
+
+use N98\Magento\Command\TestCase;
+
+class ReportCommandTest extends TestCase
+{
+    public function testExecute()
+    {
+        $this->assertDisplayRegExp(
+            array(
+                'command' => 'cache:report',
+            ),
+            '~\| ID.*\| EXPIRE.*\|$\R~m'
```

Comment:
Technically `\` requires escaping, so must be written as `\\`. `$\R` looks fishy, too as `m` modifier is used, so `\R` can be left out.